web/html/src/manager/systems/bootstrap/bootstrap-minions.tsx

```diff
@@ -196,28 +287,34 @@ class BootstrapMinions extends React.Component<Props, State> {
       (data) => {
         this.setState({
           success: data.success,
-          messages: data.messages,
+          errors: data.errors,
           loading: false,
         });
       },
       (xhr) => {
         try {
           this.setState({
             success: false,
-            messages: [JSON.parse(xhr.responseText)],
+            errors: [
+              {
+                message: JSON.parse(xhr.responseText),
+              },
+            ],
             loading: false,
           });
         } catch (err) {
-          var errMessages = DEPRECATED_unsafeEquals(xhr.status, 0)
-            ? [
-                t(
-                  "Request interrupted or invalid response received from the server. Please check if your minion was bootstrapped correctly."
-                ),
-              ]
-            : [Network.errorMessageByStatus(xhr.status)];
+          var errMessage = DEPRECATED_unsafeEquals(xhr.status, 0)
```

Comment:
If you're updating this code, please try and remove the use of `DEPRECATED_unsafeEquals`. It should most likely be `xhr.status === 0 ? ...`, but please check it.